src/frontend/Sider/Behandling/VedtakOgBeregning/Læremidler/InnvilgeVedtak/InnvilgeLæremidler.tsx

```diff
@@ -96,6 +102,9 @@ export const InnvilgeLæremidler: React.FC<{
             >
                 Lagre vedtak og gå videre
             </StegKnapp>
+            {visHarIkkeBeregnetFeilmelding && (
+                <ErrorMessage>{'Har uberenget data, vennligst ferdigstill'}</ErrorMessage>
```

Comment:
Her tenker jeg vi gjerne kan skrive en enda mer tydelig feilmelding, f.eks. 
```suggestion
                <ErrorMessage>{'Du må beregne utgiftene før du kan gå videre'}</ErrorMessage>
```

Reply:
Har gjørt